engine/sort.go

```diff
@@ -7,6 +7,7 @@ import (
 	"math"
 
 	"github.com/facette/natsort"
+
```

Comment:
nit: Some of these files contain changes that are not relevant like adding whitespace or re-ordering the imports. Could you remove them?